Dangerfile

```diff
@@ -0,0 +1,8 @@
+# Check links
+require 'json'
+results = File.read 'ab-results-README.md-markdown-table.json'
+j = JSON.parse results
+if j['error']==true
```

Comment:
[Just confirm] need `true`?

if `j['error']`'s is boolean type, you do not need to check with `true`